pkgs/development/python-modules/fpdf/default.nix

```diff
@@ -0,0 +1,19 @@
+{ stdenv, lib, writeText, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "fpdf";
+  version = "1.7.2";
+  name = "${pname}-${version}";
```

Comment:
The `name` attribute is added by `buildPython*` and should therefore be removed.

Reply:
Thanks, fixed.